### PR TITLE
BNH-48_fix feat: Names landlord on property list

### PR DESCRIPTION
**Pull request by SleepyUser**
Little fix to name the landlord whose properties are being displayed. 
image
image

Fixed thanks to Alice's feedback on the ticket while testing.

### CODE REVIEW

File: web/ViewModels/LandlordProfileModel.cs [hunk 1]
```diff
@@ -68,4 +68,23 @@ public static LandlordProfileModel FromDbModel(LandlordDbModel landlord, BricksA
...
+            Email = landlord.Email,
+            FirstName = landlord.FirstName,
+            LastName = landlord.LastName,
+            Phone = landlord.Phone,
+            Title = landlord.Title,
+            LandlordType = landlord.LandlordType,
+            LandlordProvidedCharterStatus = landlord.LandlordProvidedCharterStatus,
+            CharterApproved = landlord.CharterApproved,
+            IsLandlordForProfit = landlord.IsLandlordForProfit,
+            CurrentUserIsAdmin = landlord.User?.IsAdmin ?? false
```
Comment (BenRamchandani) [inline web/ViewModels/LandlordProfileModel.cs:87]: We should probably remove `CurrentUserIsAdmin` from LandlordProfileModel and either use `@User.HasRole` or have a separate view model for the page(s) that require it.

If you've got time to do that would you mind making a separate PR for it? If not no worries.

File: web/Controllers/LandlordController.cs [hunk 2]
```diff
@@ -191,7 +191,15 @@ public IActionResult ViewProperties(int? id = null)
 
         var databaseResult = _propertyService.GetPropertiesByLandlord(id.Value);
         var listOfProperties = databaseResult.Select(PropertyViewModel.FromDbModel).ToList();
-        return View("Properties", new PropertiesDashboardViewModel(listOfProperties));
+        var viewModel = new PropertiesDashboardViewModel(listOfProperties);
+        var landlord = await _landlordService.GetLandlordIfExistsFromId(id);
+
+        if (landlord != null)
```
Comment (BenRamchandani) [inline web/Controllers/LandlordController.cs:197]: This should never be null right? You could add another method to LandlordService `GetLandlordFromId` that throws an exception if the landlord doesn't exist. Can then make it non-null on the view model.

**Review by BenRamchandani — CHANGES_REQUESTED** [review]
One null check I think we shouldn't do

File: web/Controllers/LandlordController.cs (hunk 2 above)
Comment (SleepyUser) [reply web/Controllers/LandlordController.cs:197]: Yes, you're correct! For some reason I was thinking of the situation where an Admin who isn't a landlord tried to visit the /me/ routing. But that would be caught by the nested null check at the start of the function. 

I did consider removing the nullability from the view model and then cast what we had to be non-null to fit, but the method is more reusable.

File: web/ViewModels/LandlordProfileModel.cs (hunk 1 above)
Comment (SleepyUser) [reply web/ViewModels/LandlordProfileModel.cs:90]: I'll do a separate PR for this while Blanca is working on our editing property bits :)

**Review by BenRamchandani — APPROVED** [review]
